### DISCUSSION TITLE
How to use the themes in vscode in "npm run dev"

### DISCUSSION THREAD

**Discussion by rubenfiszel** [TypeFox/monaco-languageclient#568] [Q&A]
I'm reproducing the examples to update windmill to latest monaco-languageclient. Our project is using svelte, and it seems that for the theme to load, even in the examples, the vite config need to be updated with: https://github.com/TypeFox/monaco-languageclient/blob/main/vite.config.ts#L34.  This is incompatible with svelte own runtime loader. Would it be possible to restrict the filter so that there is less overlap with other frameworks ?

**Comment by kaisalmen** [comment]
Hi @rubenfiszel add something like this into the `onLoad` function (I pushed this change to the main branch):
```js
if (!args.path.includes('@codingame')) {
    return { contents: code };
}
```
@CGNonofr opened a [PR on vite](https://github.com/vitejs/vite/pull/14260) regarding the issue, but no real progress there.
